server/translations.go

```diff
@@ -3,8 +3,12 @@ package server
 import (
 	"context"
 	"fmt"
+	"path"
+	"path/filepath"
 	"time"
 
+	"github.com/google/go-github/v32/github"
+
 	"github.com/mattermost/mattermost-mattermod/model"
 	"github.com/mattermost/mattermost-server/v5/mlog"
```

Comment:
Nit: the import groups are mixed up. They should be arranged in the order of standard library, mattermod, and then 3rd party.